src/HRQLS/Controllers/Schools/School.php

```diff
@@ -75,8 +75,8 @@ public function main(Request $req, Application $app)
      */
     public function get(Request $req, Application $app, $id)
     {
-        //@TODO Krishna will finish implementing this.
-        return json_encode([ 'endpoint' => '/schools/{id}', 'id' => $id ]);
+        $esResult = $app['elasticsearch']->get($this->indices, $this->types, $id);
```

Comment:
This is absolutely wrong. School data is retrieved from the great schools API not Elasticsearch.